service/storage_proxy.cc

```diff
@@ -1504,6 +1504,17 @@ class abstract_write_response_handler : public seastar::enable_shared_from_this<
             timeout_cb();
         }
     }
+    void no_targets() {
+        // We don't have any live targets and we should complete the handler now.
+        // Either we already stored sufficient hints to achieve CL and the handler
+        // is completed successfuly (see hint_to_dead_endpoints), or we don't achieve
```

Comment:
Typo: `successfuly` -> `successfully`

If we merge without fixing it, this typo will haunt us from now on on other PRs...